lib/ansible/module_utils/mysql.py

```diff
@@ -34,7 +34,7 @@
     _mysql_cursor_param = 'cursor'
 except ImportError:
     try:
-        import MySQLdb as mysql_driver
+        import MySQLdb as mysql_driver, MySQLdb.cursors
```

Comment:
The sanity checks fail on this. Can you put it into a new line?
```suggestion
        import MySQLdb as mysql_driver
        import MySQLdb.cursors
```